docs/builds/Rebuild@v0.1.md

```diff
@@ -34,12 +34,16 @@ Example:
 The `externalParameters` describe the inputs to the rebuild process. This will
 be the upstream artifact, and the rebuild result.
 
-| field       | details                                                |
-| ----------- | ------------------------------------------------------ |
-| `ecosystem` | The ecosystem identifier associated with the artifact. |
-| `package`   | The package whose artifact is to be rebuilt.           |
-| `version`   | The package version whose artifact is to be rebuilt.   |
-| `artifact`  | The file name of the artifact to be rebuilt.           |
+| field                          | details                                                                                       |
+| ------------------------------ | --------------------------------------------------------------------------------------------- |
+| `ecosystem`                    | The ecosystem identifier associated with the artifact.                                        |
+| `package`                      | The package whose artifact is to be rebuilt.                                                  |
+| `version`                      | The package version whose artifact is to be rebuilt.                                          |
+| `artifact`                     | The file name of the artifact to be rebuilt.                                                  |
+| `buildConfigSource`            | The location from which the build was read. _NOTE: Only for user-generated build definitions_ |
+| `buildConfigSource.repository` | The repo URL from which the build was read.                                                   |
+| `buildConfigSource.ref`        | The repo ref from which the build was read.                                                   |
+| `buildConfigSource.path`       | The repo relpath from which the build was read.                                               |
```

Comment:
"from which the build was read" should this be "build config" or "build def" to be more precise?